docs/antora/modules/ROOT/pages/setup/docker.adoc

```diff
@@ -148,39 +167,41 @@ sudo ufw allow 9901
 
 Clone the {url_github_service}[lisk-service^] GitHub repository and then navigate into the project folder.
 
-.In the directory where you want to store Lisk Service
+.Execute in the directory where you want to store Lisk Service
 [source,bash]
 ----
 git clone https://github.com/LiskHQ/lisk-service.git
 cd lisk-service
-git checkout v0.6.4
+git checkout v0.7.0
 ----
 
 == Build Lisk Service
 
-Build Lisk Service from local files:
+To build Lisk Service from local files, first, navigate to the `lisk-service` repository on your system:
 
-./lisk-service/
+.Working directory: /lisk-service/
```

Comment:
Using root directory maybe a wrong assumption. Maybe use the relative path?
```suggestion
.Working directory: ./lisk-service/
```
or
```suggestion
.Working directory: <download_directory>/lisk-service/
```